[cob_light] missing return in uint64_t ModeExecutor::execute(cob_light::LightMode)

```
Errors     << cob_light:make /root/catkin_ws/logs/cob_light/build.make.000.log
/root/catkin_ws/src/cob_driver/cob_light/common/src/modeExecutor.cpp: In member function ‘uint64_t ModeExecutor::execute(cob_light::LightMode)’:
/root/catkin_ws/src/cob_driver/cob_light/common/src/modeExecutor.cpp:75:1: error: control reaches end of non-void function [-Werror=return-type]
 }
 ^
cc1plus: some warnings being treated as errors
```

--- a/cob_light/common/src/modeExecutor.cpp
+++ b/cob_light/common/src/modeExecutor.cpp
@@ -72,6 +72,8 @@ uint64_t ModeExecutor::execute(cob_light::LightMode requestedMode)
 	// check if mode was correctly created
 	if(mode)
 		return execute(mode);
+	else
+		return 0;
 }
 
 uint64_t ModeExecutor::execute(boost::shared_ptr<Mode> mode)

